js/index.js

```diff
@@ -0,0 +1,68 @@
+// get local storage data
+const getData = () => JSON.parse(localStorage.getItem('books'));
+
+// set local storage data
+const setData = (books) => localStorage.setItem('books', JSON.stringify(books));
+
+// reload the page
+const reloadPage = function () {
+  window.location.reload();
+  return false;
+};
+
+// Remove book
+const remove = function (index) {
+  const books = getData();
+  books.splice(index, 1);
+  setData(books);
+  reloadPage();
+};
+
+// display book
+const display = function () {
+  const books = getData();
+  const bookList = document.querySelector('#booksList');
+
+  for (let book = 0; book < books.length; book += 1) {
+    const tr = document.createElement('tr');
+
+    tr.innerHTML = `<td>${books[book].title} by ${books[book].author}</td>
+                <td><button class = "remove-book">Delete</a></td>
+                `;
+    bookList.appendChild(tr);
+  }
```

Comment:
- [ ] Good job working on the functionalities but there is a minor issue, Onl ine 26 you are looping through the books on the local storage to display them, but if on the user's browser there's no data with object name `'books'`, Local storage will return undefined and this loop will go through error because we can not check the length of undefined value. Because of this issue, your application is not running on my machine, Please make sure to fix it.
  - To fix this place your for loop in this condition `if(books) {loop}`.